packages/openai-compatible/src/openai-compatible-error.ts

```diff
@@ -23,3 +23,8 @@ export const openaiCompatibleFailedResponseHandler =
     errorSchema: openaiCompatibleErrorDataSchema,
     errorToMessage: data => data.error.message,
   });
+
+export type ErrorHandlerConfig = {
+  errorSchema: z.ZodType;
+  errorToMessage: (data: any) => string;
+};
```

Comment:
wondering about the name. it defines the json structure an error, and a way to get the message. how about `ErrorStructure` or `ErrorSchema` to give more detailed information about what it is?

Reply:
Agreed, config didn't feel right to me either, I like the edits you made.